liveHomeBackend/.env.example

```diff
@@ -0,0 +1,2 @@
+PORT=
+AUTH_JWT_SECRET=
```

Comment:
There aren't DB variables, you use it in the config/index.js

Reply:
We will add them in the next commit when configure the data base.

Reply:
Okey, for me it is approved.